# ObjectNodes.msg

Header header
int32 num_objects
ObjectNode[] objects